### PR TITLE
speed vagrant by downloading docker  images vs. build local

### PR DESCRIPTION
**Pull request by csantanapr**

### CODE REVIEW

**Comment by csantanapr** [comment]
I tested this locally and it works

File: tools/vagrant/Vagrantfile [hunk 1]
```diff
@@ -128,12 +132,12 @@ INIT_DB
...
   set -e
   set -x
-
+  echo "`date`: build-deploy-start" >> /tmp/vagrant-times.txt
   HOME=/home/vagrant
   source /etc/environment
   # Build OpenWhisk using gradle
   cd ${OPENWHISK_HOME}
-  su vagrant -c './gradlew distDocker --parallel'
+  su vagrant -c './gradlew  tools:cli:distDocker -PdockerImagePrefix=openwhisk --parallel'
```
Comment (dubeejw) [inline tools/vagrant/Vagrantfile]: I don't think we need `--parallel` here since only one project, the CLI, is being built.

**Comment by csantanapr** [comment]
I tested this and got it down to 12 minutes

**Comment by csantanapr** [comment]
With latest changes, it's working in about 15 minutes
```
vagrant@vagrant-ubuntu-trusty-64:~$ cat /tmp/vagrant-times.txt
Mon May 22 15:35:06 UTC 2017: ubuntu-setup-start
Mon May 22 15:39:51 UTC 2017: ubuntu-setup-end
Mon May 22 15:39:52 UTC 2017: couchdb-setup-start
Mon May 22 15:40:34 UTC 2017: couchdb-setup-end
Mon May 22 15:40:39 UTC 2017: build-start
Mon May 22 15:42:38 UTC 2017: deploy-start
Mon May 22 15:50:41 UTC 2017: build-deploy-end
```

**Comment by bjustin-ibm** [comment]
Will merge as soon as Travis completes.

**Pull request merged**